src/components/schedule-page/table/schedule/sections/worker-info-section/worker-info-section.component.tsx

```diff
@@ -42,14 +43,12 @@ export function WorkerInfoSection({
   );
   return (
     <div className="sectionContainer">
-      <div>
-        <NameTableComponent data={dataRows} clickable={true} {...options} />
-      </div>
-      <div>
+      <div className="sectionContainer borderContainer">
         <div>
-          <div className="table" data-cy={shiftSectionDataCy(sectionIndex)}>
-            <ShiftsSectionComponent sectionKey={options.sectionName} data={dataRows} {...options} />
-          </div>
+          <NameTableComponent data={dataRows} isWorker={true} {...options} />
+        </div>
+        <div className="table leftContainerBorder" data-cy={shiftSectionDataCy(sectionIndex)}>
+          <ShiftsSectionComponent sectionKey={options.sectionName} data={dataRows} {...options} />
```

Comment:
Pierdoła. 
Ale mi się wydaję, że te `borderContainer` potrzeba przenieść do `NamtableComponent`, `ShiftSectionComponent`,

Reply:
Chyba nie do końca rozumiem o co chodzi. Bo wprowadziłem to i wyszło tak: 
![image](https://user-images.githubusercontent.com/23723122/112302647-999db080-8c9b-11eb-9a60-b295f478410f.png)